.maintain/chaostest/src/commands/singlenodeheight/index.js

```diff
@@ -0,0 +1,61 @@
+const {Command, flags} = require('@oclif/command')
+const CONFIG = require('../../config')
+const {succeedExit, errorExit} = require('../../hypervisor/utils/exit')
+const k8s = require('../../hypervisor/deployment/k8s')
+
+class SingleNodeHeightCommand extends Command {
+  async run() {
+    const {flags} = this.parse(SingleNodeHeightCommand)
+    let port = flags.port
+    let url = flags.url
+    const wait = flags.wait || 600 * 1000
+    const height = flags.height || 10
+    const namespace = flags.namespace || CONFIG.namespace
+    const pod = flags.pod || (CONFIG.nodes && CONFIG.nodes[0])? CONFIG.nodes[0].podName: undefined
+    const jsonRpc = require('../../hypervisor/utils/json-rpc')
```

Comment:
Can you define what a `hypervisor` is in this use case? I am not quite sure I understand the name of the folder in the context of its children.